backend/conf/logback.xml

```diff
@@ -51,6 +51,9 @@
   <logger name="repositories" level="DEBUG" />
   <logger name="domain" level="DEBUG" />
   <logger name="mongo" level="DEBUG" />
```

Comment:
are the debug ones correct?

Reply:
debug once are used locally only. In all deployment we rely on the logback-prod.xml